### DISCUSSION TITLE
Some commands don't work TmT

### DISCUSSION THREAD

**Discussion by ghost** [winstxnhdw/lc-hax#574] [Q&A]
The /fakedeath /noise command doesn't work. I tested it on vanilla, but it doesn't work. Can I fix it? plz

**Comment by winstxnhdw** [comment]
What do you mean it doesn’t work? These commands have no immediate visual changes, so you’ll need to elaborate further.

**Reply by ghost** [reply]
I typed a /fakedeath command and it allows live players to see and hear me. A few months ago, when I used this command, I was recognized as dead and my opponent couldn't hear me or my voice. and The /noise command is known to provide loud noise to players, but nothing happens

**Reply by winstxnhdw** [reply]
`noise` creates a virtual noise that only enemies can hear. Not you or the player. `fakedeath` only works if you die after you use the command.

**Reply by ghost** [reply]
Oh no! I didn't know how to use it. Thank you for letting me know.

**Reply by ghost** [reply]
I'm sorry. What is Fakedeath? Is it dead and alive?

**Reply by winstxnhdw** [reply]
IIRC it kills yourself without actually killing yourself. So everyone else but you thinks you are dead.

**Reply by ghost** [reply]
hello I used the fakedeath command today, but it didn’t falsely make me appear dead. everyone still thought I was alive, they could see me, and they could hear my voice. After using the command, I tried to commit suicide, but I couldn’t do anything. I can’t figure out the conditions under which it works I’m sorry to bother you.

**Reply by ghost** [reply]
Oh, and the "open" and "close" commands don’t open or close the ship doors. I just thought I’d let you know about this as well.

**Reply by winstxnhdw** [reply]
I’ll take a look at it again. It’s been a while since I used it.

**Reply by ghost** [reply]
Thank you so much!
